data/cy/census_individual_gb_wls.json

```diff
@@ -9222,16 +9476,10 @@
                                                 "id": "employers-business-answer",
                                                 "label": "Disgrifiad",
                                                 "mandatory": true,
-                                                "max_length": 200,
-                                                "type": "TextArea",
-                                                "validation": {
-                                                    "messages": {
-                                                        "MAX_LENGTH_EXCEEDED": "Mae\u2019n rhaid i\u2019ch ateb fod yn llai na %(max)d o nodau"
-                                                    }
-                                                }
+                                                "type": "TextField"
                                             }
                                         ],
-                                        "description": "Er enghraifft, gwerthu dillad, ysbyty cyffredinol, addysg gynradd, gwerthu bwyd i fusnesau, y gwasanaeth sifil (Llywodraeth Cymru), llywodraeth leol (tai).",
+                                        "description": "Er enghraifft, manwerthu dillad, ysbyty cyffredinol, addysg gynradd, cyfanwerthu bwyd, y gwasanaeth sifil (Llywodraeth Cymru), llywodraeth leol (tai).",
```

Comment:
I think we might have inadvertently reverted to an older version of this translation. I am pretty sure we previously corrected this text so that it matched the old version shown in this diff.

Reply:
Hmm.. the file was taken from the other pr. I'm investigating with @iwootten.